backend/Makefile

```diff
@@ -10,7 +10,7 @@ run:
 	./${BINARY_NAME}
 
 test:
-	go test -v -cover ./...
+	go test -v ./tests
```

Comment:
Not sure if this will work, since I have added unit tests in the strategies directory. `./tests` contains integration based tests, but we may have unit tests in the other directories. Unless we want to structure the unit tests differently.